1.yaml

```diff
@@ -0,0 +1,11 @@
+Parameters:
+        UnusedParameter:
```

Comment:
cfn-lint: W2001 Parameter UnusedParameter not used.  <!-- Automatically posted by Lintly -->